game_patch/multi/server.cpp

```diff
@@ -305,6 +316,37 @@ static void send_private_message_with_stats(rf::Player* player)
     send_chat_line_packet(str.c_str(), player);
 }
 
+void shuffle_level_array()
+{
+    std::ranges::shuffle(rf::netgame.levels, rng);
+    xlog::warn("Shuffled level rotation");
+}
+
+const char* get_rand_level_filename()
+{
+    std::size_t num_levels = rf::netgame.levels.size();
```

Comment:
```suggestion
    const std::size_t num_levels = rf::netgame.levels.size();
```
What is not used  mutably should be `const`.